test/mocks/admin.js

```diff
@@ -0,0 +1,31 @@
+// eslint-disable-next-line
```

Comment:
What is the error with the linter? You can identify the rule that is failing and deactivate it for the entire file, so you don't need to disable single lines.

Reply:
The error is associated with parameters that must be used.
I use mocks from the answers functions but I must keep the parameter structure input, so the code can behave as it should.